astropy/io/ascii/core.py

```diff
@@ -1440,12 +1440,10 @@ def read(self, table):
                 # otherwise, we raise an error only if it is still inconsistent
                 if len(str_vals) != n_cols:
                     errmsg = (
-                        "Number of header columns ({}) inconsistent with"
-                        " data columns ({}) at data line {}\n"
-                        "Header values: {}\n"
-                        "Data values: {}".format(
-                            n_cols, len(str_vals), i, [x.name for x in cols], str_vals
-                        )
+                        f"Number of header columns ({n_cols}) inconsistent with"
+                        f" data columns ({len(str_vals)}) at data line {i}\n"
```

Comment:
```suggestion
                        f"Number of header columns ({n_cols}) inconsistent with "
                        f"data columns ({len(str_vals)}) at data line {i}\n"
```
Surprised how some `.format`s have still survived to this day.
Just little nit-pick to add to follow general line-break convention.